Formula/gcc.rb

```diff
@@ -172,19 +174,7 @@ def install
     # Even when suffixes are appended, the info pages conflict when
     # install-info is run. TODO fix this.
     info.rmtree
-
-    # Rename java properties
-    if build.with?("java") || build.with?("all-languages")
-      config_files = [
-        "#{lib}/gcc/#{version_suffix}/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/security/classpath.security",
-        "#{lib}/gcc/#{version_suffix}/i386/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/i386/security/classpath.security",
-      ]
-      config_files.each do |file|
-        add_suffix file, version_suffix if File.exist? file
-      end
-    end
+    # Since GCC 4.9 java properties are properly sandboxed.
```

Comment:
It [looks like](https://github.com/Homebrew/homebrew-core/blame/master/Formula/gcc.rb#L176-L186) it was first added by you ( @mikemcquaid ) June 6th, 2014 to gcc.rb in https://github.com/Homebrew/homebrew-core/commit/9cc81cee476e5d956afa2f0d58820200848a2fb1

@manphiz [removed it](https://github.com/Homebrew/homebrew-versions/blame/master/gcc6.rb#L180) in the initial addition of homebrew-versions/gcc6.rb back in January 2016